src/toil/common.py

```diff
@@ -94,15 +94,15 @@ def __init__(self):
         self.maxNodes = [10]
         self.targetTime = defaultTargetTime
         self.betaInertia = 0.1
-        self.scaleInterval = 30
+        self.scaleInterval = 60
         self.preemptableCompensation = 0.0
         self.nodeStorage = 50
         self.metrics = False
 
         # Parameters to limit service jobs, so preventing deadlock scheduling scenarios
         self.maxPreemptableServiceJobs = sys.maxsize
         self.maxServiceJobs = sys.maxsize
-        self.deadlockWait = 60  # Wait one minute before declaring a deadlock
+        self.deadlockWait = 60  # Wait 80 seconds before declaring a deadlock
```

Comment:
Again, comment seems to be referring to a different value. (Maybe we should just rewrite it to "number of seconds to wait before declaring a deadlock"?)

Reply:
Thanks for catching this too.